wework/src/components/chat/composer/ModelSelector.module.css

```
@@ -4,10 +4,15 @@
 }
 
 .submenu {
-  transform-origin: left bottom;
+  transform-origin: var(--radix-popover-content-transform-origin);
+  --model-selector-submenu-enter-x: -7px;
   animation: model-selector-submenu-enter 170ms cubic-bezier(0.2, 0, 0, 1) both;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)